metadata/shortlinks.yaml

```diff
@@ -1363,6 +1367,13 @@ id:
   S00111: "/topics/single-cell/tutorials/scrna-trajectories/slides.html"
   F00352: "/faqs/gtn/gtn_licenses.html"
   N00058: "/news/2023/07/20/prefs.html"
+  T00363: "/topics/contributing/tutorials/fair-gtn/tutorial.html"
+  T00364: "/topics/imaging/tutorials/bioimage-REMBI/tutorial.html"
+  T00365: "/topics/imaging/tutorials/bioimage-metadata/tutorial.html"
+  T00366: "/topics/single-cell/tutorials/scrna-case_FilterPlotandExploreRStudio/tutorial.html"
+  F00353: "/topics/single-cell/faqs/single_cell_omics.html"
+  F00354: "/topics/single-cell/faqs/user_community_join.html"
+  N00059: "/news/2023/10/05/wendi.html"
```

Comment:
Should this one really be called wendi? Maybe embeddable_widgets?

Reply:
it appears only in the URL, not the titlebar. you'll notice it's not the only news item with a less than explicit filename. Please feel free to PR it if you feel strongly.